### DISCUSSION TITLE
Enhanced table formatting

### DISCUSSION THREAD

**Discussion by stereo9** [outline/outline#3712] [Ideas]
Hi,

my team found out that while the dynamic table formatting is epic, used in combination with images it can create quite a messy table layout. it would be cool to include enhanced table formatting options, like fixed cell/col/row sizes.

Also some functions don't work within the tables, like #3066

**Comment by stereo9** [comment]
After much more testing, the following points could be very helpful

- Task list inside a table #3066 
- bullet points inside a table
- text alignment left / right / center inside a cell
- adjustable col width
- merge cells

**Reply by chenyf** [reply]
hi, what's the progress of "merge cells" ?

**Comment by IvanTahoe** [comment]
I'd like to second this idea as well - being able to format tables would be a tremendous help!

**Comment by UKFr-DIZ** [comment]
Instead of developing yourself, it should be considered to integratable tables, which have already implemented the rich editor functionality: 
like from Google, onedrive etc. or i guess this https://tabulator.info/examples/5.4 for example :)
